docs/name-owners/atomic-name-trading/index.md

```diff
@@ -0,0 +1,294 @@
+
+```
+$ namecoind decoderawtransaction <large hex string>
+{
+  "txid" : "92a50d2495d66cbfb13ae87deaec059ab605a7bf737b10935f9e0f80065debac",
+  "version" : 28928,
+  "locktime" : 0,
+  "vin" : [
+    {
+      "txid" : "12ed9f83b83e248e2ca19e0197933914013145981fcacc5ba47c27e60eacbfaa",
+      "vout" : 0,
+      "scriptSig" : {
+        "asm" : "304502202131c98c98a3dfe19713ec0e48917a502063028766abf292b46fb8e8a96d80df022100f22198c86e814eff3a2562a36de5249a425c08080cb174eace7da9529e6e16b801",
+        "hex" : "48304502202131c98c98a3dfe19713ec0e48917a502063028766abf292b46fb8e8a96d80df022100f22198c86e814eff3a2562a36de5249a425c08080cb174eace7da9529e6e16b801"
+      },
+      "value" : 974.90000000,
+      "sequence" : 4294967295
+    },
+    {
+      "txid" : "00abd8282558574369038722d24c65355e1b98ca3f00d7d09cb7bea8732b598a",
+      "vout" : 1,
+      "scriptSig" : {
+        "asm" : "",
+        "hex" : ""
+      },
+      "value" : 0.01000000,
+      "sequence" : 4294967295
+    }
+  ],
+  "vout" : [
+    {
+      "value" : 10.00000000,
+      "n" : 0,
+      "scriptPubKey" : {
+        "asm" : "OP_DUP OP_HASH160 3c4b7d4b93bc6194087bbbc422fd6cd1a40f820e OP_EQUALVERIFY OP_CHECKSIG",
+        "hex" : "76a9143c4b7d4b93bc6194087bbbc422fd6cd1a40f820e88ac",
+        "reqSigs" : 1,
+        "type" : "pubkeyhash",
+        "addresses" : [
+          "mm1mGfLEMjFPyGsbkcoYiN6yiPcBztK6Dw"
+        ]
+      }
+    },
+    {
+      "value" : 964.89500000,
+      "n" : 1,
+      "scriptPubKey" : {
+        "asm" : "OP_DUP OP_HASH160 ab96c9bfb7c55de825be7f12bd38dd25b62be02b OP_EQUALVERIFY OP_CHECKSIG",
+        "hex" : "76a914ab96c9bfb7c55de825be7f12bd38dd25b62be02b88ac",
+        "reqSigs" : 1,
+        "type" : "pubkeyhash",
+        "addresses" : [
+          "mwAEUoA3NzSmASyWxrTK1mhsd4pFvruoXj"
+        ]
+      }
+    },
+    {
+      "value" : 0.01000000,
+      "n" : 2,
+      "scriptPubKey" : {
+        "nameOp" : {
+          "op" : "name_update",
+          "name" : "d/my-cool-domain",
+          "value" : "thanks from Sally"
+        },
+        "asm" : "NAME_OPERATION OP_DUP OP_HASH160 ad4a0929e9c7c95910534b93ec0727058a27f2b9 OP_EQUALVERIFY OP_CHECKSIG",
+        "hex" : "5310642f6d792d636f6f6c2d646f6d61696e117468616e6b732066726f6d2053616c6c796d7576a914ad4a0929e9c7c95910534b93ec0727058a27f2b988ac",
+        "reqSigs" : 1,
+        "type" : "pubkeyhash",
+        "addresses" : [
+          "mwKDtLZ9EHSeL8TjkZAcppsnb7M3LRhoGR"
+        ]
+      }
+    }
+  ],
+  "fees" : 0.00500000
+}
+```
+
+It is also crucial to check which inputs Bob has prepared for her to sign, so that she doesn't end up sending something she doesn't intend to send. I'm not completely sure what would be the easiest and safest way to ensure that the transaction from Bob is not malicious. A possible way to guard against this could be to look up the address holding the sold name, export its private key, and use "signrawtransaction" with this private key as optional argument.
```

Comment:
Maybe suggest here to use `gettxout` and check that all the inputs (except for the name itself) have addresses *not* owned by Sally.

Reply:
Any chance you could submit a follow-up PR for this?

Reply:
Yeah sure, also this can be done in a follow up.